\documentclass[11pt]{article}

\usepackage{geometry}
\geometry{left=1.1in, right=1.1in, top=1.0in, bottom=0.95in}

% graphicx package, useful for including eps and pdf graphics
\usepackage{graphicx}
\DeclareGraphicsExtensions{.pdf,.png,.jpg}

% basic packages
\usepackage{color}
\usepackage{parskip}
\setlength{\parskip}{0.16cm}
\usepackage{float}
\usepackage{todonotes}
\usepackage{enumitem}
\usepackage{microtype}

\definecolor{green}{rgb}{0.20,0.50,0.48}
\usepackage[hidelinks]{hyperref}
\hypersetup{colorlinks=true,linkcolor=black,citecolor=black,urlcolor=green}

\setlength{\parindent}{0pt} % Remove paragraph indent

\definecolor{brown}{rgb}{0.700,0.150,0.150}
\def\mfc#1{\textcolor{brown}{[#1]}}

\begin{document}

\thispagestyle{empty} % Remove page headers/footers

\mbox{}\hfill
\begin{tabular}{l @{}}
	\includegraphics[width=6.5cm]{figures/fhcrc_logo} \\
	Vaccine and Infectious Disease Division \\
	Fred Hutchinson Cancer Research Center \\
	1100 Fairview Ave N \\
	Seattle, WA 98109, USA \\
	Phone: (206) 667-6372 \\
	Email: \href{mailto:mfiggins@uw.edu}{mfiggins@uw.edu} \\
\end{tabular}

\vspace{0.1in} % Vertical skip between sender/receiver address

\begin{tabular}{@{} l}
  \today
\end{tabular}

\vspace{0.1in} % Vertical skip between receive address and letter opening

Dear Editors,

\medskip % Vertical skip between letter opening and letter body

Please find attached our manuscript entitled ``Inferring variant-specific effective reproduction numbers from combined case and sequencing data''.
We would be grateful if you considered it for publication in \textit{eLife}.

Much of the COVID-19 pandemic was characterized by waves of infection driven by the repeated emergence of SARS-CoV-2 variants.
This has led to development of several models which estimate the selective advantage of SARS-CoV-2 variants using frequencies of given variants in sequenced data (for example \href{https://www.nature.com/articles/s41586-021-03908-2}{Annavajhala et al.\ 2021} and \href{https://www.science.org/doi/10.1126/science.abm1208}{Oberymeyer et al.\ 2022}).

Though these methods are useful, most rely on population genetic models which often assume constant pathogen population sizes, constant growth advantages for variants and do not quantify variant transmission dynamics directly due to several simplifying implicit assumptions.
This is complicated by the fact that various biological, social, and political phenomena such as immune escape, population structure, and non-pharmaceutical interventions can contribute to changing fitness landscapes for respiratory viruses.
Understanding this variation in growth advantages over time and across locations is therefore important for understanding both the evolution and transmission dynamics of SARS-CoV-2 more generally.

In order to address how variant growth advantages differ over time and between locations, we develop a model of variant dynamics where variant frequencies are determined directly by a transmission model.
This allows us to determine a growth advantage for variants at the level of the time-varying effective reproduction number and estimate absolute growth rates for each variant.
We apply this model to 1.5M sequenced viral genomes to jointly estimate the variant-specific effective reproductive numbers, growth advantages, and frequencies in 34 US states for 8 SARS-CoV-2 variants.
We show that variant growth advantages are relatively consistent between states with Omicron exhibiting the highest variation between states and there is strong evidence for temporal variation in variant growth advantages.
Further, we relate the observed variant incidence and growth advantages to estimates of past exposure and vaccination as a way to highlight various mechanisms which may underlie heterogeneity in variant growth advantages.
Our results suggest that the inclusion of factors relating population immunity and exposure may be useful for further quantification and explanation of SARS-CoV-2 variant dynamics and epidemic potential.

In 2022, we applied this model to provide continually updated estimates of spread of Omicron and Omicron sublineages BA.2, BA.2.12.1, BA.4 and BA.5 and shared results in real-time to public health partners, policy makers and the public through \href{https://github.com/blab/rt-from-frequency-dynamics}{GitHub} and \href{https://twitter.com/trvrb/status/1530649628625936384}{Twitter}.
As SARS-CoV-2 continues to evolve, it will be critical to have accurate and robust methods to estimate variant-specific growth advantages and transmission to inform ongoing pandemic response and inform vaccine strain selection.
As case surveillance has declined, we note that our work can also be applied to multiple different data sources such as hospitalizations or potentially wastewater.

Most importantly, we believe that this represents a general framework for assessing variant-specific Rt and could be applied to other pathogens such as mpox (whereever we have distinct lineages contributing to an epidemic).
This manuscript is relevant to a general audience including evolutionary biologists, infectious disease researchers and computational biologists.
For this reason, we would appreciate your consideration for publication in \textit{eLife}.

In a candid side note: this work was strongly motivated by SARS-CoV-2 pandemic response and having variant-specific Rt results in a timely fashion.
We believe it served this purpose well.
However, we've been slow to publish this work and as such some of the manuscript framing doesn't fully match today's reality (with lack of systematic case reporting).
Still, we believe this a strong general method and although manuscript results represent a snapshot into 2021 and 2022, it's still a useful historical picture of what occurred in the initial variant-driven epidemic waves.

\vspace{0.3in} % Vertical skip between letter closing and signature start

Sincerely, \newline
\vspace{0.05in} \newline
Marlin Figgins \newline
PhD Candidate \newline
Department of Applied Mathematics, University of Washington \newline
Vaccine and Infectious Disease Division, Fred Hutchinson Cancer Center

\vspace{0.2in}

Trevor Bedford \newline
Professor, Vaccine and Infectious Disease Division, Fred Hutchinson Cancer Center \newline
Investigator, Howard Hughes Medical Institute \newline
Affiliate Professor, Departments of Genome Sciences and Epidemiology, University of Washington

\end{document}
